corehq/apps/app_manager/tests/data/suite/suite-case-detail-tabs-with-nodesets-for-sorting-search-only.xml

```diff
@@ -70,7 +70,7 @@
             </sort>
           </field>
         </detail>
-        <detail nodeset="instance('commtrack:products')/some/data">
+        <detail nodeset="instance('commtrack:products')/some/data" relevant="gender='male'">
```

Comment:
@Charl1996 What's this for?

Reply:
A fix for the `test_case_detail_tabs_with_nodesets_for_sorting_search_only_field` test, since the feature is now unrestricted.